.github/workflows/translation_check.py

```diff
@@ -148,8 +150,11 @@ def count_translations(dir_with_translations, print_info, reference_file):
             print(f'\t{', '.join(messages["missing"])}')
             print(f'Unnecessary (not in en-GB) ({not_needed[lang]}):')
             print(f'\t{', '.join(messages["unnecessary"])}')
-            print(f'Unexpected (should not be translated) ({len(messages["unexpected"])}):')
+            print(f'Technical strings modified ({len(messages["unexpected"])}):')
             print(f'\t{', '.join(messages["unexpected"])}')
+            if len(messages["unexpected"]):
```

Comment:
Could be just:
```suggestion
            if messages["unexpected"]:
```
No?

Reply:
🐍👍 

Yes, that is more pythonic, thanks!